### PR TITLE
Fast R-CNN

### PR DESCRIPTION
**Pull request by jodoin**

### CODE REVIEW

File: deep-learning/_posts/2017-05-14-fastrcnn.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+tags:   deep-learning CNN localization 
+author: Pierre-Marc Jodoin 
+pdf:   "https://arxiv.org/pdf/1504.08083.pdf"
+cite:
+  authors: "R.Girshick"
+  title:   "https://arxiv.org/pdf/1504.08083.pdf"
+  venue:   "International Conference on Computer Vision ({ICCV})"
+---
+
+This is an effort to improve [R-CNN]({{ site.baseurl }}{% link deep-learning/_posts/2017-04-18-rcnn.md %}).  Instead of feeding prospective bounding boxes to a CNN+SVM, the input image is fed to a VGG16 net.  At the **conv5** layer, each bounding box is used iteratively to crop the feature maps, followed by an **ROI Pooling** layer in order to resize the croped feature maps.  This is followed by the usual **FC layers + softmax**.  A bounding box regressor is also used to repositioned the bounding boxes.
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-05-14-fastrcnn.md]: reposition

File: deep-learning/_posts/2017-05-14-fastrcnn.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+<div align="middle">
+  <img src="/deep-learning/images/fastrcnn/sc01.jpg" width="500">
+</div>
+
+<br>
+While results are slighly better than R-CNN on VOC 2017/10/12, Fast R-CNN is much faster both at train time and at test time.
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-05-14-fastrcnn.md]: slightly

File: deep-learning/_posts/2017-05-14-fastrcnn.md (hunk 1 above)
Comment (lemairecarl) [inline deep-learning/_posts/2017-05-14-fastrcnn.md]: >each bounding box is used iteratively

What is iterating here?

File: deep-learning/_posts/2017-05-14-fastrcnn.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+layout: review
+title:  "Fast R-CNN"
+tags:   deep-learning CNN localization 
+author: Pierre-Marc Jodoin 
+pdf:   "https://arxiv.org/pdf/1504.08083.pdf"
+cite:
+  authors: "R.Girshick"
+  title:   "https://arxiv.org/pdf/1504.08083.pdf"
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-05-14-fastrcnn.md]: Why didn't you put the title here?

File: deep-learning/_posts/2017-05-14-fastrcnn.md (hunk 1 above)
Comment (jodoin) [inline deep-learning/_posts/2017-05-14-fastrcnn.md]: I meant sequentially

**Pull request merged**